goalie/go_mesh_seq.py

```diff
@@ -222,6 +224,54 @@ def indicators(self):
             self._create_indicators()
         return self._indicators
 
+    def _remove_adjoint_labels(self):
+        r"""
+        Remove adjoint-related solution labels and associated fields from the
+        solution dictionary.
+        """
+        for fieldname in self.field_functions:
+            if self.steady:
+                self.solutions.labels = ("forward",)
+                self.solutions[fieldname].pop("forward_old", None)
+            else:
+                self.solutions.labels = ("forward", "adjoint")
+                self.solutions[fieldname].pop("forward_old", None)
+                self.solutions[fieldname].pop("adjoint_next", None)
```

Comment:
```suggestion
                self.solutions[fieldname].pop("forward_old")
            else:
                self.solutions.labels = ("forward", "adjoint")
                self.solutions[fieldname].pop("forward_old")
                self.solutions[fieldname].pop("adjoint_next")
```
`None` is not necessary here and by passing that argument we are preventing errors from being raised. I.e., `self.solutions[fieldname].pop("forward_old")` will raise an error if `"forward_old"` key is not found (it would instead just return `None`). So I think it's good to remove it, so that errors are raised if something goes wrong.